addons/hr_holidays/models/hr_leave.py

```diff
@@ -235,6 +257,19 @@ def unlink(self):
     def copy_data(self, default=None):
         raise UserError(_('A leave cannot be duplicated.'))
 
+    @api.model
+    def default_get(self, fields_list):
```

Comment:
Guideline note: default_get definition should be above the fields definition, with other default methods.